Aztec/Classes/TextKit/TextView.swift

```diff
@@ -635,9 +634,9 @@ open class TextView: UITextView {
     ///   - text: the text being added
     ///   - range: the range of the insertion of the new text
     ///
-    private func refreshBlockquoteAfterInsertionOf(text: String, range:NSRange) {
+    private func refreshBlockquoteAfterInsertion(of text: String, at range: NSRange) {
         let formatter = BlockquoteFormatter()
-        guard formatter.attribute(inTextView: self, at: range.location) else {
+        guard formatter.present(in: textStorage, at: range.location) else {
```

Comment:
Unless I'm missing something, we could change this line to become:

```swift
guard formattingAtIndexContainsBlockquote(range.location) else {
```

Which would also allow us to remove the formatter from this method.